### PR TITLE
Fixes #6417, fixes being able to pick up paperbins through windows

### LINKED ISSUE #6417: Paperbin in backpack: Have to remove all the papers one by one
You can click+drag a paperbin to yourself, which puts it in your hand. You can then carry it around and put it in your backpack.
If you put it in your backpack, though, you can't put it back in your hand. Clicking it inside your backpack only takes one paper from it. You can't click+drag it to your hand slot like you can do with a backpack, either.

And that's awful

### PR DESCRIPTION
**Pull request by 9600bauds**
fixes #6417
performs the autoclose on #6417
technically not a feature technically not freeze

### CODE REVIEW

**Comment by Cyadox** [comment]
The best PRs are the ones that fix issues that aren't even reported :+1:

**Comment by Probe1** [comment]
it works perfectly fine you goon, you click drag it onto your mob not your hand

**Comment by 9600bauds** [comment]
This makes it so that you can clickdrag into either

File: code/modules/paperwork/paperbin.dm [hunk 1]
```diff
@@ -23,14 +23,23 @@
...
-	if((user == usr && (!( usr.restrained() ) && (!( usr.stat ) && (usr.contents.Find(src) || in_range(src, usr))))))
+/obj/item/weapon/paper_bin/MouseDrop(over_object)
+	if(!usr.restrained() && !usr.stat && (usr.contents.Find(src) || Adjacent(usr)))
 		if(!istype(usr, /mob/living/carbon/slime) && !istype(usr, /mob/living/simple_animal))
-			if( !usr.get_active_hand() )		//if active hand is empty
-				src.loc = user
-				user.put_in_hands(src)
-				user.visible_message("<span class='notice'>[user] picks up the [src].</span>", "<span class='notice'>You grab [src] from the floor!</span>")
-
+			if(istype(over_object,/obj/screen)) //We're being dragged into the user's UI...
```
Comment (d3athrow) [inline code/modules/paperwork/paperbin.dm]: if you're going to check the name don't bother with a type check and just do a null sanity check

Comment (9600bauds) [inline code/modules/paperwork/paperbin.dm]: i tried to do that and it said "over_object.name" didn't exist for some reason

**Pull request merged**